docs/how-to/push-update-on-mainnet.md

```diff
@@ -183,44 +183,42 @@ depends on whether you want to update `cardano-sl` or `csl-daedalus`.
+stack exec -- cardano-auxx $COMMONOPTS $AUXXOPTS cmd --commands "propose-update 0 vote-all:true 0.1.0 ~software~cardano-sl:1"
 ```
 
 You also can combine importing keys and sending an update proposal in
 one command for your convenience. Example:
 
 ```
-stack exec -- cardano-auxx $COMMONOPTS $AUXXOPTS cmd --commands "add-key key0.sk primary, add-key key1.sk primary, add-key key2.sk primary, add-key key3.sk primary, propose-update 0 vote-all 0.1.0 65536 70000 csl-daedalus:1 win64 ${WIN64_INSTALLER} none macos64 ${DARWIN_INSTALLER} none"
+stack exec -- cardano-auxx $COMMONOPTS $AUXXOPTS cmd --commands "add-key ./key0.sk primary:true; add-key ./key1.sk primary:true; add-key ./key2.sk primary:true; add-key ./key3.sk primary:true; propose-update 0 vote-all:true 0.1.0 bvm ~software~csl-daedalus:1 (upd-bin "win64" ${WIN64_INSTALLER}) (upd-bin "macos64" ${DARWIN_INSTALLER})"
 ```
 
 Let's break down the invocation of `propose-update`. First come arguments that you almost certainly won't need to modify:
 
 * `0` is the index of key that will be used to sign the update.
-* `vote-all` flag means that update proposal will be submitted along
+* `vote-all:true` flag means that update proposal will be submitted along
   with votes from all our keys.
 * `0.1.0` is block version to be used after the
   update. See [Prerequisites](#prerequisites), you should know it in
   advance. `0.1.0` is used as an example.
-* `65536` is maximal tx size. Should be set to this value (`65536`) in `0.1.0`.
-* `70000` is maximal size of update proposal. Should be set to this
-  value (`70000`) in `0.1.0`.
 
-The next argument (`<software-version>`, e. g. `csl-daedalus:1`) is
+The next argument (`~software~<software-version>`, e. g. `~software~csl-daedalus:1`) is
 software version description. You should substitute `1` (version) with
 the integer provided along with installers (see *Prerequisites*
 section). Recall that currently we are maintaining two softwares:
 `cardano-sl` (used by core and relay nodes) and `csl-daedalus` (used
 by wallets).
 
 Further arguments are optional, should be provided in `csl-daedalus`
-case and omitted in `cardano-sl` case. Those arguments are an arbitrary number
-of _triples_ (two in `csl-daedalus` case, one for Windows and one for
-macOS). Each triple is `<system tag> <installer filename> none`. Once
-again, those are filenames in the current dir, not arbitrary
-paths. `none` stands for binary diff (this feature is not used for
-now). System tags should corresponds to `systemTag` values in
-configuration file.
+case and omitted in `cardano-sl` case. Those arguments are update proposals
+for a system. You can specify an arbitary amount of them
+(two in `csl-daedalus` case, one for Windows and one for macOS).
+They're constructed with `upd-bin` function, which takes two arguments:
```

Comment:
`upd-bin` takes three arguments, two of them optional

Reply:
I know, I thought it is better to not overcomplicate things and not mention argument that is not actually used at the moment (according to the previous version of the documentation).